alma/alma-woocommerce-gateway
# Blocks display deferred days

## Reason for change


Display correctly with design the pnx block.

[Linear task](https://linear.app/almapay/issue/ECOM-1403/[💅-wc-blocks]-deferred-payments-blocks-are-displayed)

### Code changes



### How to test

_As a reviewer, you are encouraged to test the PR locally._
On woocommerce 8-5-1 or 8-6-0, on the checkout page, you should be able to see the new block.
![image](https://github.com/alma/alma-woocommerce-gateway/assets/43644110/1fd38f9a-9c89-4829-bda4-19d5c06b7e81)




### Checklist for authors and reviewers




### Non applicable
